14/nonickid/log_command_line.py

```diff
@@ -0,0 +1,38 @@
+from functools import wraps
+from getpass import getpass
+
+username = None
+password = None
+
+def logcommand(func):
+    '''Decorator to ...'''
+    @wraps(func)  # preserves function meta data
+    def wrapper(*args, **kwargs):
+        # do stuff before func
+        cmd = func(*args, **kwargs)
+        # do stuff after func
+        if len(cmd) > 0:
+           print(f"Command '{cmd}' executed by {username} has been logged")
```

Comment:
You might consider using the built in logging module or Logbook rather than print.